scripts/dockerhost/start-dockerhosts

```diff
@@ -1,11 +1,40 @@
 #!/bin/bash
 
+set -e
+
 scriptdir=`dirname "$BASH_SOURCE"`
 echo $scriptdir
 
+if ! which nsenter 2>&1 >/dev/null
+then
+  echo "Please install nsenter. It's a part of the util-linux package, which may not be current on your system."
+  exit 1
+fi
+
+if ! sudo ip link | grep -q docker0
+then
```

Comment:
@jainvipin 
I think this condition arises today because of this script, if ovs is not installed on the host and the script is run then it ends up wiping the docker0 interface of the host. I think this check is trying to catch and help that condition. @erikh kindly confirm in case I missed something.

Just as a side note, one of the pre-req checks added in #65 checks for ovs being installed on the host and shall help prevent this condition from occurring as well.